lib/licenses.nix

```diff
@@ -30,6 +30,13 @@ in mkLicense lset) ({
     fullName = "Abstyles License";
   };
 
+  acsl14 = {
+    fullName = "Anti-Capitalist Software License v1.4";
+    url = "https://anticapitalist.software/";
+    free = false;
+    redistributable = true;
```

Comment:
I don't think this license is redistributable — it doesn't make any distinction between use and distribution in its conditions, so if you're not allowed to use it, you're also not allowed to redistribute it.

Reply:
@alyssais 
I do not know the nuances of what counts as "redistributable" in nixpkgs conventions, so I copied the entry for the Hippocratic License 3.0 (licenses.hl3), as the closest analogue in the repo.

If the criterion above holds for the ACSL, shouldn't it hold for the HL too? As I understand it, if you don't comply with the ethical standards outlined in HL section 3, you lose all "rights granted to the Licensee", which by 1.3 include the right to distribution.

Reply:
Yes, I think you're right — the Hippocratic License shouldn't be marked as redistributable either.  It looks like the addition of the license wasn't closely reviewed at the time.